cmd/snap/cmd_known.go

```diff
@@ -31,11 +31,25 @@ import (
 	"github.com/jessevdk/go-flags"
 )
 
+type assertTypeName string
+
+func (n assertTypeName) Complete(match string) []flags.Completion {
+	names := asserts.TypeNames()
```

Comment:
bit confused, wasn't the idea to use client.AssertionTypes here? it's not needed otherwise